File: src/main/java/pl/arturczopek/mycoach/repository/UserSettingRepository.java
package pl.arturczopek.mycoach.repository;

import org.springframework.data.repository.CrudRepository;
import pl.arturczopek.mycoach.model.database.UserSetting;

/**
 * @Author Artur Czopek
 * @Date 17-02-2017
 */
public interface UserSettingRepository extends CrudRepository<UserSetting, Long> {

}

File: src/main/kotlin/pl/arturczopek/mycoach/repository/UserSettingRepository.kt
package pl.arturczopek.mycoach.repository

import org.springframework.data.jpa.repository.Query
import org.springframework.data.repository.CrudRepository
import pl.arturczopek.mycoach.model.database.UserSettings

/**
 * @Author Artur Czopek
 * @Date 30-06-2017
 */

interface UserSettingRepository: CrudRepository<UserSettings, Long> {

    @Query("select us.infoMail from UserSettings us")
    fun findAllEmails(): List<String>
}